patches/react-native-web+0.19.9+001+initial.patch

```diff
@@ -1,286 +1,648 @@
 diff --git a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
-index c879838..288316c 100644
+index c879838..0c9dfcb 100644
 --- a/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
 +++ b/node_modules/react-native-web/dist/vendor/react-native/VirtualizedList/index.js
-@@ -117,6 +117,14 @@ function findLastWhere(arr, predicate) {
-  *
-  */
- class VirtualizedList extends StateSafePureComponent {
-+  pushOrUnshift(input, item) {
```

Comment:
Commenting to cross-reference this in another issue. I believe this PR caused [this](https://github.com/Expensify/App/issues/37447)